tests/test_image.py

```diff
@@ -69,7 +69,7 @@ def makeRampImage(width, height, imgClass=afwImage.ImageF):
     val = 0
     for yInd in range(height):
         for xInd in range(width):
-            im.set(xInd, yInd, val)
+            im[xInd, yInd, afwImage.PARENT] = val
```

Comment:
see comment in `test_background.py`. There's no need to include the origin since `PARENT` is the default value.